tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,40 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+
+        cv = make_content_view({
+            u'organization-id': self.org['id']
+        })
+        repo = self._make_file_repository_upload_contents({
+            u'content-type': 'file',
+            u'organization-id': self.org['id'],
+            u'product-id': self.product['id']
+        })
+        ContentView.add_repository({
+            u'id': cv['id'],
+            u'repository-id': repo['id'],
+            u'organization-id': self.org["id"]})
+        env = make_lifecycle_environment({
+            u'organization-id': self.org['id']
+        })
+        ContentView.publish({
```

Comment:
that is fine to be in one line as string has less than 99 symbols:
`ContentView.publish({u'id': cv['id']})`